### PR TITLE
BorrowBookUi ready for review

### PR DESCRIPTION
**Pull request by jazzie-b**
Refer to commit no. 2, dfbea84 for actual code changes

### CODE REVIEW

**Comment by TwistedHead** [comment]
All code style guidelines have been followed. I am OK for it to be merged with the Master.

**Comment by pyroib** [comment]
These don't meet the style guides. please review again, add all faults as per style guides and allow programmer to resubmit

File: src/BorrowBookUi.java [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+						input("Press <any key> to complete ");
+					}
+					break;
+				case COMPLETED:
+					output("Borrowing Completed");
+					return;
+				default:
+					output("Unhandled state");
+					throw new RuntimeException("BorrowBookUi : unhandled state :" + state);			
+				}
```
Comment (pyroib) [inline src/BorrowBookUi.java]: check indentation for the end of the switch and while statements

File: src/BorrowBookUi.java [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+	}
+
+	
+	private String input(String prompt) {
+		System.out.print(prompt);
+		return input.nextLine();
+	}	
+		
+		
+	private void output(Object object) {
```
Comment (pyroib) [inline src/BorrowBookUi.java]: usage of method names that are broad or that could cover multiple actions can cause problems. 

'output' can mean multiple thing. Consider 'printOutput', 'printToScreen' or something similar

File: src/BorrowBookUi.java [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+	
+	public BorrowBookUi(BorrowBookControl control) {
+		this.control = control;
+		input = new Scanner(System.in);
+		state = UiState.INITIALISED;
+		control.setUi(this);
+	}
+
+	
+	private String input(String prompt) {
```
Comment (pyroib) [inline src/BorrowBookUi.java]: usage of method names that are broad or that could cover multiple actions can cause problems. 

'input' can mean multiple thing. Consider 'userInput', 'getUserInput' or something similar

**Comment by jazzie-b** [comment]
Changed made, please review.

**Pull request merged**